### ISSUE TITLE
[Bug]: MPRIS doesn't work correctly

### ISSUE THREAD

**Issue by Totto16** [th-ch/youtube-music#1755]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.1

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official YTM web version](https://music.youtube.com).

### What operating system are you using?

Ubuntu

### Operating System Version

Ubuntu 23.10

### What CPU architecture are you using?

x64

### Last Known Working YouTube Music (Application) version

3.2.2

### Reproduction steps

Update the application to 3.3.1 and restart it

### Expected Behavior

MPRIS works normally, as it did before in v3.2.2 (and still does)

### Actual Behavior

MPRIS show up in status bar for a few millis seconds, after that it dissapears and 
```bash
playerctl status
```
says: `No players found`

### Enabled plugins

tuna-obs::backend 
quality-changer::backend 
music-together::backend 
in-app-menu::backend 
sponsorblock::backend 
shortcuts::backend 
lyrics-genius::backend 
adblocker::backend 
"shortcuts::menu" 
"lyrics-genius::menu" 
"in-app-menu::menu" 
"downloader::menu" 
"album-color-theme::menu" 
"adblocker::menu" 
"shortcuts::menu" 
"lyrics-genius::menu" 
"in-app-menu::menu" 
"downloader::menu" 
"album-color-theme::menu" 
"adblocker::menu" 

### Additional Information

Errors during startup:
```
[165277:0218/175231.222211:ERROR:nss_util.cc(357)] After loading Root Certs, loaded==false: NSS error code: -801

Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:962:11)
    at Socket._write (node:net:974:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Socket.Writable.write (node:internal/streams/writable:337:10)
    at EventEmitter.e.message (/opt/YouTube Music/resources/app.asar/dist/main/index.js:198:2938)
    at Lke.send (/opt/YouTube Music/resources/app.asar/dist/main/index.js:185:1849)
    at EventEmitter.r (/opt/YouTube Music/resources/app.asar/dist/main/index.js:181:1806) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
Error: Cannot send message, stream is closed
    at EventEmitter.e.message (/opt/YouTube Music/resources/app.asar/dist/main/index.js:198:2730)
    at Lke.send (/opt/YouTube Music/resources/app.asar/dist/main/index.js:185:1849)
    at EventEmitter.n (/opt/YouTube Music/resources/app.asar/dist/main/index.js:181:1297)
    at EventEmitter.emit (node:events:517:28)
    at Function.emitPropertiesChanged (/opt/YouTube Music/resources/app.asar/dist/main/index.js:177:34633)
    at i.setProperty (/opt/YouTube Music/resources/app.asar/dist/main/index.js:201:16284)
    at Ar.set [as playbackStatus] (/opt/YouTube Music/resources/app.asar/dist/main/index.js:205:6789)
    at Ar.setPlaybackStatus (/opt/YouTube Music/resources/app.asar/dist/main/index.js:205:7714)
    at /opt/YouTube Music/resources/app.asar/dist/main/index.js:205:10316
    at IpcMainImpl.<anonymous> (/opt/YouTube Music/resources/app.asar/dist/main/index.js:174:25180)
Checking for update
Error: Cannot send message, stream is closed
    at EventEmitter.e.message (/opt/YouTube Music/resources/app.asar/dist/main/index.js:198:2730)
    at Lke.send (/opt/YouTube Music/resources/app.asar/dist/main/index.js:185:1849)
    at EventEmitter.r (/opt/YouTube Music/resources/app.asar/dist/main/index.js:181:1806)
    at EventEmitter.emit (node:events:517:28)
    at i.e.descriptor.value (/opt/YouTube Music/resources/app.asar/dist/main/index.js:177:33039)
    at Ar.seeked (/opt/YouTube Music/resources/app.asar/dist/main/index.js:205:4724)
    at IpcMainImpl.<anonymous> (/opt/YouTube Music/resources/app.asar/dist/main/index.js:205:8627)
    at IpcMainImpl.emit (node:events:517:28)
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:77582)
    at WebContents.emit (node:events:517:28)
Error: Cannot send message, stream is closed
    at EventEmitter.e.message (/opt/YouTube Music/resources/app.asar/dist/main/index.js:198:2730)
    at Lke.send (/opt/YouTube Music/resources/app.asar/dist/main/index.js:185:1849)
    at EventEmitter.r (/opt/YouTube Music/resources/app.asar/dist/main/index.js:181:1806)
    at EventEmitter.emit (node:events:517:28)
    at i.e.descriptor.value (/opt/YouTube Music/resources/app.asar/dist/main/index.js:177:33039)
    at Ar.seeked (/opt/YouTube Music/resources/app.asar/dist/main/index.js:205:4724)
    at /opt/YouTube Music/resources/app.asar/dist/main/index.js:205:10304
    at IpcMainImpl.<anonymous> (/opt/YouTube Music/resources/app.asar/dist/main/index.js:174:25389)


```

**Comment by Totto16** [comment]
After some investigating, If found out, that 845711510581f4603ec863825404321b15d5a444 alias the bug fix to #1726 is the cause of this problem

**Comment by Totto16** [comment]
I can confirm, that #1760 fixes it locally 👍🏼 ❤️
